silk/unit_tests/test_log_replay.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+import queue
+import unittest
+
+from silk.tests import silk_replay
+from silk.tools.otns_manager import OtnsManager
+from silk.unit_tests.mock_service import MockGrpcClient, MockUDPServer
+from silk.unit_tests.testcase import SilkTestCase
+
+
+class OTNSLogReplayTest(SilkTestCase):
+    """Silk unit test case for log replayer.
+    """
+
+    def setUp(self) -> None:
+        """Test method set up.
+        """
+        self.exception_queue = queue.Queue()
+
+        self.manager = OtnsManager("localhost", self.logger.getChild("OtnsManager"))
+        self.grpc_client = MockGrpcClient(self.exception_queue, self.logger.getChild("MockGrpcClient"))
+        self.manager.grpc_client = self.grpc_client
+
+        self.udp_server = MockUDPServer(self.exception_queue)
+
+        hwconfig_path = Path(__file__).parent / "fixture/hwconfig.ini"
+        self.args = ["tests/silk_replay.py", "-v2", "-c", str(hwconfig_path), "-s", "localhost", "-p", "100"]
```

Comment:
Is speed of 100 too much ?

Reply:
If the speed is lower the unit tests are going to take longer. I think 100 is an appropriate setting.